neo/rawio/neuralynxrawio/nlxheader.py

```diff
@@ -115,6 +115,12 @@ def _to_bool(txt):
             filename_regex=r"## File Name: (?P<filename>\S+)",
             datetimeformat="%m/%d/%Y %H:%M:%S",
         ),
+        "peg": dict(
+            datetime1_regex=r"-TimeCreated (?P<date>\S+) (?P<time>\S+)",
+            datetime2_regex=r"-TimeClosed (?P<date>\S+) (?P<time>\S+)",
+            filename_regex=r'-OriginalFileName "?(?P<filename>\S+)"?',
+            datetimeformat="%Y/%m/%d %H:%M:%S.%f",
```

Comment:
The test seems to be failing because you are asking for fractional seconds. But one of our current test data sets doesn't have this fraction. Are you sure that these datasets all have a fractional second provided. or sometimes they don't. In which case this may need to be different @Antho2422 ?

Reply:
The problem was that Pegasus v2.2.2 uses different time formats in the opening and closing timestamps. PR#1408 fixes this and should replace this PR. I suggest closing this one for the time being and merging that one.